report/report_style

```diff
@@ -0,0 +1,37 @@
+<style>
+        body {
+            background-color: #2a2a2a;
+            color: #f5f5f5;
+            font-family: Arial, sans-serif;
```

Comment:
No need to change the whole thing, but just a reminder that if you want the colours to match the dashboard at any point, you can look up the colour codes in the dashboard folder.